applications/vasp/Performance Study 2/cpu-bind data/README.md

```diff
@@ -0,0 +1,10 @@
+The cpu-bind_VASP.ipynb looks at how tasks bind to cores using various cpu-bind settings on Swift and Eagle. It is independent of the cpu-bind recommendations in the Performance Study 2 documentation, and instead serves on only to better understand the direct effect of the cpu-bind flag on the way tasks are processed when running VASP on Swift and Eagle. It uses the data in the eagle_onnodes_data and swift_onnodes_data folders. 
```

Comment:
I think the "on" needs to be taken out or changed here "instead serves on only to better understand"

Reply:
Ok, fixed it!